validator/src/commands/run/args.rs

```diff
@@ -3,7 +3,8 @@ use {
         cli::{hash_validator, port_range_validator, port_validator, DefaultArgs},
         commands::{FromClapArgMatches, Result},
     },
-    clap::{App, Arg, ArgMatches},
+    clap::{values_t, App, Arg, ArgMatches},
+    indexmap::IndexSet,
```

Comment:
This data structure may actually be better in the long run; the below snippet isn't something to be proud of but can we limit changes like this. Given that these PR's verge on large, I'd prefer we come back and do cleanup like that after the fact:
```rust
        .collect::<Result<HashSet<_>, _>>()?
        .into_iter()
        .collect::<Vec<_>>();
 ```

Reply:
I’m using indexmap because it preserves input order, which makes it easier to write tests. I could get it back to the old logic, but that will invovle some hacks in the shared test functions too. if you don't feel strongly about it, maybe we can sneak this change 🙈